pumps/splunk.go

```diff
@@ -199,6 +205,27 @@ func (p *SplunkPump) WriteData(ctx context.Context, data []interface{}) error {
 					continue
 				}
 
+				// Check if the current analytics field is "tags" and see if some tags are explicitly excluded
+				if (field == "tags" && len(p.config.IgnoreTagPrefixList) > 0) {
+					filteredTags := mapping["tags"].([]string)
+
+					// Loop all explicitly ignored tags
+					for _, excludeTag := range p.config.IgnoreTagPrefixList {
+						// Loop the current analytics item tags
+						for key, currentTag := range filteredTags {
+							// If the current tag's value includes an ignored word, remove it from the list
+							if strings.Contains(currentTag, excludeTag) {
```

Comment:
@mbelchev  It should use `strings.HasPrefix` instead of contains.

Reply:
@tbuchaillot, yep, agree. It's now adjusted, thanks!